.github/workflows/rake.yml

```diff
@@ -11,18 +10,28 @@ name: rake
     branches:
       - master
 jobs:
-  rake:
+  test:
```

Comment:
@rdavid it's better to have same names for the job and the workflow, for consistency